plugins/jobs/web_client/views/JobStatusSegmentizer.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'underscore';
+import JobStatus from '../JobStatus';
+
+export default class JobStatusSegmentizer {
+    segmentize(jobs) {
```

Comment:
@jbeezley, I see. Moving it into JobListWidget to indicate that it's used as a helper. I that case, the timeChartConfig phaseChartConfig file problem should be moved too. Will creating a subdirectory for all three helper file for the JobList help?